### PR TITLE
Added additional functionallity to the minimal response to retain mor…

### PR DESCRIPTION
**Pull request by Robert430404**
…e context without the need to have a full response

### CODE REVIEW

**Comment by Robert430404** [comment]
I'll get the CI violations addressed and update the pull request.

**Comment by shaunburdick** [comment]
Thanks, I'll take a look. Honestly I haven't updated this in a while and it could probably use a rewrite.

File: config.default.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ const config = {
     token: 'xoxb-Your-Token',
     autoReconnect: true
   },
+  disablePretext: true,
+  disableFootnote: true,
```
Comment (shaunburdick) [inline config.default.js:25]: Talk to me about why you'd want to disable the footer?

Comment (Robert430404) [reply config.default.js:25]: It was a request from my director and the owner of the company I work at, they wanted to slim to payload as much as possible while still retaining some additional information in the minimal view.

**Comment by Robert430404** [comment]
I'm working on finding out why the unit tests fail. It's something to do with the node version not liking the syntax of the requires for the bot test. If I run it on the latest LTS all tests pass no problem.

**Pull request closed without merging**